src/plugins/mf_priority.cpp

```diff
@@ -331,6 +331,25 @@ static json_t *user_to_json (
 }
 
 
+// Scan the users map and look at each user's default bank to see if any one
+// of them have a valid bank (i.e one that is not "DNE"; if any of the users do
+// do have a valid bank, it will return false)
+static bool check_map_for_dne_only ()
+{
+    // check if the map only contains DNE entries
+    bool only_dne_data = true;
```

Comment:
This variable appears to be unused

Reply:
Whoops, thanks for catching this. I must have missed that when I was moving this block of code to a helper function. Will fix this